core/spec.md

```diff
@@ -3670,6 +3725,16 @@ references an attribute in that entity (i.e. there is no `PATH`), then if the
 `EXPRESSION` does not match the entity, that entity MUST NOT be returned. In
 other words, a `404 Not Found` would be generated in the HTTP protocol case.
 
+At this time, this specification only supports filtering over scalar attributes
+defined at the root of the xRegistry entities (the Registry itself, Groups,
+Resources and Versions). The one exception to this are maps (such as `labels`)
```

Comment:
The singular/plural clash here is bugging me.
```suggestion
Resources and Versions). The one exception to this is support for filtering on maps (such as `labels`)
```

Reply:
done